sdk/monitor/monitor-query/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 2.0.0-beta.1 (2023-11-08)
```

Comment:
This should just be a minor version update as there are no breaking changes - `1.2.0-beta.1`

Reply:
Done. Will Update in the next commit